tests/conftest.py

```diff
@@ -0,0 +1,84 @@
+from copy import deepcopy
+import os
+from pathlib import Path
+import shutil
+from typing import Callable, Dict, Iterator, Optional, Union
+
+import pytest
+
+from everest.config.control_config import ControlConfig
+
+
+@pytest.fixture(scope="session")
+def testdata() -> Path:
+    return Path(__file__).parent / "test_data"
+
+
+@pytest.fixture
+def copy_testdata_tmpdir(
+    testdata: Path, tmp_path: Path
+) -> Iterator[Callable[[Optional[str]], Path]]:
+    def _copy_tree(path: Optional[str] = None):
+        path_ = testdata if path is None else testdata / path
+        shutil.copytree(path_, tmp_path, dirs_exist_ok=True)
+        return path_
+
+    cwd = Path.cwd()
+    os.chdir(tmp_path)
+    yield _copy_tree
+    os.chdir(cwd)
+
+
+@pytest.fixture(scope="module")
+def control_data_no_variables() -> Dict[str, Union[str, float]]:
+    return {
+        "name": "group_0",
+        "type": "well_control",
+        "min": 0.0,
+        "max": 0.1,
+        "perturbation_magnitude": 0.005,
+    }
+
+
+@pytest.fixture(
+    scope="module",
```

Comment:
Why is this a module scope?

Reply:
This is only used in test_controls.py, maybe move it to that test file.

Reply:
Yes, these newly created fixture is only used a handful of times. I struggle to get the tests working with `relpath` function  and `tmpdir` decorator. I asked for help a couple of weeks back, in the meantime, I used the power of `pytest` `fixtures` and `parameterization`. I am all for a unified system, and would welcome a revert to `relpath` and `tmpdir`.

Reply:
It's ok for now not critical!